tests/test_core.py

```diff
@@ -137,6 +138,14 @@ class TestComponents:
         "name": {"type": "string", "example": "doggie"},
     }
 
+    # Referenced examples are only supported in OAS 3.x
+    @pytest.mark.parametrize("spec", ("3.0.0",), indirect=True)
+    def test_example(self, spec):
+        spec.components.example("PetExample", {"value": {"a": "b"}})
+        defs = get_examples(spec)
+        assert "PetExample" in defs
```

Comment:
Nitpick ++

I'd remove this assert since it is covered by the line below.

I can do that later if you don't have time for that.

Reply:
Done